lib/ops_manager_ui_drivers/wait_helper.rb

```diff
@@ -2,7 +2,14 @@
 
 module OpsManagerUiDrivers
   module WaitHelper
-    SLEEP_INTERVAL = 0.5
+
+    def self.sleep_interval=(sleep_interval)
```

Comment:
WaitHelper is being mixed in, therefore these methods cannot be class methods. You can see an example usage in `ops_manager_ui_drivers/version17/available_products.rb`.